CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+* CI: Setup Symlinks from the inner build to the outer Gradle wrapper. (#146)
```

Comment:
```suggestion
```

I'd just set _#skip-changelog_ in the PR description, this is not an user-facing change

Reply:
Yup makes sense 👍